app/views/shared/_checkbox_group.html.haml

```diff
@@ -5,9 +5,10 @@
       %input.govuk-input.search-input__input{ placeholder: "Search" }
       %button{type: "submit"}
         %i.fa.fa-search
-  = f.govuk_collection_check_boxes attribute_name,
-    options,
-    value_method,
-    text_method,
-    hint_method = hint_method,
-    small: small
+  %div{ class: "#{classes.present? ? classes : ''}" }
```

Comment:
You can give the classes attribute to the FormBuilder object itself, so no need for this line